pytests/test_bjoern.py

```diff
@@ -0,0 +1,464 @@
+# -*- coding: utf-8 -*-
+
+import contextlib
+import json
+import math
+import os
+import unittest
+import signal
+import socket
+import sys
+import textwrap
+from wsgiref.validate import validator
+
+import pytest
+import requests
+import bjoern
+
+
+@pytest.mark.parametrize('header_x_auth', ('X-Auth_User', 'X_Auth_User', 'X_Auth-User'))
+def test_CVE_2015_0219(client, header_x_auth):
+    """
+    https://www.djangoproject.com/weblog/2015/jan/13/security/
+    https://snyk.io/vuln/SNYK-PYTHON-BJOERN-40507
+    """
+
+    def app(env, start_response):
+        assert header_x_auth not in env.keys()
+        assert 'X-Auth-User' not in env.keys()
```

Comment:
Shouldn't these headers be transformed to `HTTP_X_AUTH_USER` etc (since WSGI servers do that before putting them to env)

Reply:
Oh man, this is so embarrassing. Thanks for spotting this!